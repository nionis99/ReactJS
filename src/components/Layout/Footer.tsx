import React, { HTMLAttributes } from 'react';
import AppLogo from 'components/Logo';

const Footer = ({ className = '', ...restProps }: HTMLAttributes<HTMLDivElement>) => (
  <header className={`flex items-center justify-center bg-footer w-full h-17.5 ${className}`} {...restProps}>
    <AppLogo />
  </header>
);

export default Footer;
