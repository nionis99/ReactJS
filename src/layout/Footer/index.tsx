import React, { HTMLAttributes } from 'react';
import AppLogo from 'components/Logo';

const Footer = ({ className = '', ...restProps }: HTMLAttributes<HTMLDivElement>) => (
  <footer className={`flex items-center justify-center bg-footer w-full py-6 ${className}`} {...restProps}>
    <AppLogo />
  </footer>
);

export default Footer;
