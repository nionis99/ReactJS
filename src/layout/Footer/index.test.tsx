import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '.';

it('should render footer component', () => {
  const elem = renderer.create(<Footer />).toJSON();
  expect(elem).toMatchSnapshot();
});
