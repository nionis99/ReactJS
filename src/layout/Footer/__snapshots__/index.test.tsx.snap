// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render footer component 1`] = `
<footer
  className="flex items-center justify-center bg-footer w-full py-6 "
>
  <test-file-stub
    height={24}
    width={150}
  />
</footer>
`;
